### PR TITLE
error handling of register and login forms, proxy port changed as same as flask default port

### PR DESCRIPTION
**Pull request by Tushar19varshney**
The proxy port is now 5000 same as Flask default port

Fixed the validations for login and register forms like:
ezgif com-video-to-gif (1)
ezgif com-video-to-gif (2)

Fixes #204

### CODE REVIEW

**Comment by Tushar19varshney** [comment]
@ivantha @root5533 Please review:slightly_smiling_face:

File: fact-bounty-client/package.json [hunk 1]
```diff
@@ -31,7 +32,7 @@
     "lint": "eslint src/**/*.js",
     "format": "prettier --write \"src/**/*.{js,jsx,json}\""
   },
-  "proxy": "http://localhost:7000",
+  "proxy": "http://localhost:5000",
```
Comment (ivantha) [inline fact-bounty-client/package.json]: What is the reason for this change?
